pkgs/top-level/python-packages.nix

```diff
@@ -5868,7 +5868,7 @@ in {
       sha256 = "1v9j9fmf8g911yg6k01xa2db6dx3wv73zkk7fncsj7vagjqgs20l";
     };
 
-    propagatedBuildInputs = with self; [ self.six ];
+    propagatedBuildInputs = with self; [ six ];
```

Comment:
Please remove this unrelated change as it can cause trouble in the future with cherry-picking.